pom.xml

```diff
@@ -220,7 +220,10 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <exclude>com.google.cloud.spanner.IntegrationTest</exclude>
+          <excludes>
+            <exclude>com.google.cloud.spanner.jdbc.it.**</exclude>
+            <exclude>com.google.cloud.spanner.jdbc.JdbcStatementTimeoutTest</exclude>
```

Comment:
Why did we exclude the tests under it/ and the new test that we added this in PR (JdbcStatementTimeoutTest)?

Reply:
The integration tests should already have been excluded, but the existing exclude didn't actually work (it references a `Category` that is not used). The reason that the integration tests weren't executed by the `maven-surefire-plugin` was because of the default shared configuration that excluded them. So in theory we could remove the exclude here. But as it was already present and intentional, but not working, I decided to fix it so it does work.

The reason that we are excluding the `StatementTimeoutTest` here is that it is relatively slow. I felt that it was overkill to execute the test every time you execute `mvn test`, which is already taking quite long for this library, and instead only run it together with the integration tests. So the test is executed by the CI, but not locally unless you explicitly enable it.